### PR TITLE
[Documentation] factory boy

### PR DESCRIPTION
**Pull request by a-c-caeiro**
Factory boy does not have the best documentation IMO
This a practical "How to use" factory boy with Django models

### CODE REVIEW

**Review by kenvontucky — APPROVED** [review]

**Review by hugo-serra — APPROVED** [review]

File: factory_boy.md [hunk 1]
```diff
@@ -0,0 +1,132 @@
...
+        model = Band
+        django_get_or_create = ("name",)
+    
+class MusicTrackFactory(models.Model):
+    name = factory.Faker('name')
+    band = factory.SubFactory(BandFactory)
+    release_date = factory.Faker('date_this_century',before_today=True)
+    duration = fake.random_int(min=30, max=500)
+    class Meta:
+        model = MuiscTrack
```
Comment (unstablectrl) [inline factory_boy.md]: Not that it matters much but `Music` is misspelled in a couple of these examples

File: onboarding/factory_boy.md [hunk 2]
```diff
@@ -0,0 +1,132 @@
...
+Factory boy has many ways to generate data, usually we prefer to use Faker, 
+factory boy provides a wrapper for faker 
+
+### Simple faker attribute
+````python
+class MusicTrackFactory(models.Model):
+    name = factory.Faker('name')
+
+    class Meta:
+        model = MuiscTrack
```
Comment (a-c-caeiro) [inline onboarding/factory_boy.md]:
```suggestion
        model = MusicTrack
```

File: onboarding/factory_boy.md [hunk 3]
```diff
@@ -0,0 +1,132 @@
...
+    class Meta:
+        model = Band
+        django_get_or_create = ("name",)
+    
+class MusicTrackFactory(models.Model):
+    name = factory.Faker('name')
+    band = factory.SubFactory(BandFactory)
+
+    class Meta:
+        model = MuiscTrack
```
Comment (a-c-caeiro) [inline onboarding/factory_boy.md]:
```suggestion
        model = MusicTrack
```

File: onboarding/factory_boy.md [hunk 4]
```diff
@@ -0,0 +1,132 @@
...
+        model = Band
+        django_get_or_create = ("name",)
+    
+class MusicTrackFactory(models.Model):
+    name = factory.Faker('name')
+    band = factory.SubFactory(BandFactory)
+    release_date = factory.Faker('date_this_century',before_today=True)
+    duration = fake.random_int(min=30, max=500)
+    class Meta:
+        model = MuiscTrack
```
Comment (a-c-caeiro) [inline onboarding/factory_boy.md]:
```suggestion
        model = MusicTrack
```

**Review by a-c-caeiro — COMMENTED** [review]
a

**Pull request merged**